Framework/PythonInterface/plugins/algorithms/ComputeCalibrationCoefVan.py

```diff
@@ -0,0 +1,168 @@
+from mantid.api import PythonAlgorithm, AlgorithmFactory, WorkspaceProperty, Progress, InstrumentValidator
+from mantid.kernel import Direction
+import mantid.simpleapi as api
+import numpy as np
+from scipy import integrate
+import scipy as sp
+import mlzutils
```

Comment:
Yes a separate pull request with the refactoring changes will be fine. Yes a separate module containing the do_guassian_fit would probably be best.